docs/reference/settings/data-frames-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+
+[role="xpack"]
+[[data-frames-settings]]
+=== {dataframe-transforms-cap}  settings in Elasticsearch
+++++
```

Comment:
Asciidoctor doesn't like attributes in this type of titleabbrev unless you add the following:
```suggestion
[subs="attributes"]
++++
```